zerver/lib/test_fixtures.py

```diff
@@ -178,39 +182,36 @@ def check_file_hash(target_file_path: str, status_dir: str) -> bool:
 
     return _check_hash(source_hash_file, target_content)
 
-def check_setting_hash(setting_name: str, status_dir: str) -> bool:
+def check_setting_hash(status_dir: str) -> bool:
+    setting_name = 'REALM_INTERNAL_BOTS'
```

Comment:
Not a good move IMO. The function was created with the idea of abstraction.
That check_settings dir in `template_database_status` exists so that we can just add new settings to be checked for changes.

Reply:
Updated. Thanks for the review.